extensions/investspec/test/test_module/test_module/test_module.py

```diff
@@ -0,0 +1,196 @@
+import pint
+
+ureg = pint.UnitRegistry()
+ureg.define('none = []')
+
+ARGS_SPEC = {
+    "model_name": "Forest Carbon Edge Effect Model",
+    "module": __name__,
+    "userguide_html": "carbon_edge.html",
+    "args_with_spatial_overlap": {
+        "spatial_keys": ["aoi_vector_path", "lulc_raster_path"],
+    },
+    "args": {
+        "number_input": {
+            "name": "Foo",
+            "about": "Numbers have units that are displayed in a human-readable way.",
+            "type": "number",
+            "units": ureg.meter**3/ureg.month,
+            "expression": "value >= 0"
+        },
+        "ratio_input": {
+            "name": "Bar",
+            "about": "Here's a ratio.",
+            "type": "ratio"
+        },
+        "percent_input": {
+            "name": "Baz",
+            "about": "Here's a percent.",
+            "type": "percent",
+            "required": False
+        },
+        "code_input": {
+            "name": "Abc",
+            "about": "Here's a code.",
+            "type": "code",
```

Comment:
I found myself thinking the same thing reading through the tests.

Reply:
Looking back at it I totally agree... especially since we decided to allow negative integers, "integer" is really the best description. I'll make a PR changing "code" to "integer" in natcap/invest.

Reply:
invest PR: https://github.com/natcap/invest/pull/589

Reply:
Just merged it.